src/Vault.sol

```diff
@@ -199,7 +199,6 @@ contract Vault {
      */
     function openTrustedMarginAccount(address protocol) public onlyOwner {
         require(!isTrustedCreditorSet, "V_OMA: ALREADY SET");
-        //ToDo: Check in Factory/Mainregistry if protocol is indeed trusted?
 
         (bool success, address baseCurrency_, address liquidator_) =
             ITrustedCreditor(protocol).openMarginAccount(vaultVersion);
```

Comment:
make sure interface is a view function, otherwise this is an open door

Reply:
Good catch, should make a find the bug out of this for Twitter clout